docs/pages/setup/reference/backends.mdx

```diff
@@ -214,7 +214,7 @@ teleport:
 The AWS authentication settings above can be omitted if the machine itself is
 running on an EC2 instance with an IAM role.
 
-The optional `GET` parameters shown below control how Teleport interacts with an S3 endpoint, including S3-compatible endpoints.
+The optional `GET` parameters shown below are teleport-specific options to configure how Teleport interacts with an S3 endpoint, including S3-compatible endpoints.
```

Comment:
I think we can make this sentence clearer by restructuring it with Teleport as the subject, maybe with something like:

```md
You can add optional query parameters to the S3 URL. The Teleport Auth
Service reads these parameters to configure its interactions with S3.
```